com.unity.render-pipelines.universal/Runtime/UniversalAdditionalCameraData.cs

```diff
@@ -337,18 +348,61 @@ public void SetRenderer(int index)
             m_RendererIndex = index;
         }
 
+        /// <summary>
+        /// Returns the selected scene-layers affecting this camera.
+        /// </summary>
         public LayerMask volumeLayerMask
         {
             get => m_VolumeLayerMask;
             set => m_VolumeLayerMask = value;
         }
 
+        /// <summary>
+        /// Returns the transform that will act as a trigger for volume blending.
+        /// </summary>
         public Transform volumeTrigger
         {
             get => m_VolumeTrigger;
             set => m_VolumeTrigger = value;
         }
 
+        /// <summary>
+        /// Returns true if this camera requires to color information in a texture.
+        /// If enabled, color texture is available to be bound and read from shaders as _CameraOpaqueTexture after rendering skybox.
+        /// </summary>
+        public bool requiresVolumeFrameworkUpdate
+        {
+            get
+            {
+                if (m_VolumeFrameworkUpdateModeOption == VolumeFrameworkUpdateModeOverrideOption.UsePipelineSettings)
+                {
+                    return UniversalRenderPipeline.asset.volumeFrameworkUpdateMode == VolumeFrameworkUpdateMode.EveryFrame;
+                }
+                else
+                {
+                    return m_VolumeFrameworkUpdateModeOption == VolumeFrameworkUpdateModeOverrideOption.EveryFrame;
+                }
+            }
+            set => m_VolumeFrameworkUpdateModeOption = (value) ? VolumeFrameworkUpdateModeOverrideOption.EveryFrame : VolumeFrameworkUpdateModeOverrideOption.ViaScripting;
+        }
+
+        /// <summary>
+        /// Returns the current volume stack used by this camera
+        /// </summary>
+        VolumeStack m_VolumeStack = null;
+        public VolumeStack volumeStack
```

Comment:
(As you know but replying in case other reviewers are thinking the same)
We now have functions on the Camera to update the volume framework. 
If users would like to change the stack used by this camera, then they have the possibility here to replace the volumeStack with their custom one.